lib/active_model/serializer/associations.rb

```diff
@@ -65,6 +65,16 @@ def has_one(name, options = {}, &block) # rubocop:disable Style/PredicateName
           associate(HasOneReflection.new(name, options, block))
         end
 
+        # Set _default_include to the parsed value of +include+.
+        # @param includes value to be parsed by JSONAPI::IncludeDirective::Parser
+        # @param options options for JSONAPI::IncludeDirective::Parser, default { allow_wildcard: true }
+        # @return [void]
+        #
+        def default_include(include, options = {})
+          default_options = { allow_wildcard: true }
+          self._default_include = JSONAPI::IncludeDirective.new(include, default_options.merge(options))
```

Comment:
Actually, changed to `include_args` to follow JSONAPI::IncludeDirective#initialize convention 😄